flytekit/common/tasks/sdk_runnable.py

```diff
@@ -447,6 +447,8 @@ def _get_container_definition(
                 "{{.input}}",
                 "--output-prefix",
                 "{{.outputPrefix}}",
+                "--raw-data-output-prefix",
+                "{{.outputPrefix}}"
```

Comment:
This should be raw output prefix, right?

Reply:
yeah this is when i realized that flyteplugins wasn't done